ruby_event_store/lib/ruby_event_store/specification.rb

```diff
@@ -33,14 +33,36 @@ def from(start)
     # Limits the query to events before or after another event.
     # {http://railseventstore.org/docs/read/ Find out more}.
     #
-    # @param start [String] id of event to start reading from.
+    # @param stop [String] id of event to start reading from.
     # @return [Specification]
     def to(stop)
       raise InvalidPageStop if stop.nil? || stop.empty?
       raise EventNotFound.new(stop) unless reader.has_event?(stop)
       Specification.new(reader, result.dup { |r| r.stop = stop })
     end
 
+    # Limits the query to events before or after another event.
+    # {http://railseventstore.org/docs/read/ Find out more}.
+    #
+    # @param date [Date]
```

Comment:
Why `Date`? Why not `Time` ? It will be a common scenario to filter events only from a very short amount of time (minutes or hours) - not always full days are the range we want to get while debugging some problems.

Reply:
Good point. I will change. This is the kind of feedback I wanted before diving deeper.